docs/source/models.rst

```diff
@@ -299,8 +405,22 @@ automatic dependency management).
 Once loaded, you can score the model by calling the :py:func:`predict <mlflow.pyfunc.PyFuncModel.predict>`
 method, which has the following signature::
 
-  predict(model_input: pandas.DataFrame) -> [numpy.ndarray | pandas.(Series | DataFrame)]
-
+  predict(model_input: [pandas.DataFrame, numpy.ndarray, Dict[str, np.ndarray]]) -> [numpy.ndarray | pandas.(Series | DataFrame)]
+  
+All model flavors will support pandas.DataFrame as input and some will also support tensor inputs in the form
+of numpy.ndarrays.
```

Comment:
I think we should mention that generally DL flavors are expected to support tensors? We should also tell users to check documentation for the particular flavor to verify whether it supports tensor input.